Neuron/Sections/Guide/GuideService.swift

```diff
@@ -43,11 +43,17 @@ class GuideService {
         guard controller == nil else { return }
         let guideController: GuideViewController = UIStoryboard(name: .guide).instantiateViewController()
         controller = BaseNavigationController(rootViewController: guideController)
+        controller?.modalPresentationStyle = .overCurrentContext
         UIApplication.shared.keyWindow?.rootViewController?.present(controller!, animated: true, completion: nil)
     }
 
     private func hideGuide() {
-        controller?.dismiss(animated: true, completion: nil)
-        controller = nil
+        UIView.animate(withDuration: 0.33, animations: {
```

Comment:
Hint: we can use `CATransaction.animationDuration` to get the default system animation duration.